src/lib/core-extension-service.ts

```diff
@@ -79,14 +79,34 @@ export class CoreExtensionService {
 	 *
 	 * @returns The {@link TelemetryService} object exported by the Core Extension if available, else null.
 	 */
-	public static getTelemetryService(): TelemetryService|null {
+	public static _getTelemetryService(): CoreTelemetryService|null {
```

Comment:
The leading underscore is a JS social convention indicating that the method should be treated by developers as private even though it's technically public.